Formula/l/ldid-procursus.rb

```diff
@@ -0,0 +1,26 @@
+class LdidProcursus < Formula
+  desc "Put real or fake signatures in a Mach-O binary"
+  homepage "https://github.com/ProcursusTeam/ldid"
+  url "https://github.com/ProcursusTeam/ldid.git",
+     tag:      "v2.1.5-procursus7",
+     revision: "aaf8f23d7975ecdb8e77e3a8f22253e0a2352cef"
+  version "2.1.5-procursus7"
+  license "AGPL-3.0-or-later"
+  head "https://github.com/ProcursusTeam/ldid.git", branch: "master"
+
+  depends_on "pkg-config" => :build
+  depends_on "libplist"
+  depends_on "openssl@3"
+
+  conflicts_with "ldid", because: "ldid installs a conflicting ldid binary"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+    zsh_completion.install "_ldid"
+  end
+
+  test do
+    cp test_fixtures("mach/a.out"), testpath
+    system "#{bin}/ldid", "-S", "a.out"
```

Comment:
```suggestion
    system bin/"ldid", "-S", "a.out"
```

Reply:
This seems like a very weird nitpick, considering that [other formulas do it the way I have](https://github.com/Homebrew/homebrew-core/blob/master/Formula/d/dos2unix.rb#L43). I'm open to just change it the current state is non-negotiable.

Reply:
I'm sure they do, but whenever possible we prefer avoiding string interpollation.